src/components/questions/questionsList.js

```diff
@@ -10,8 +10,21 @@ constructor(props){
   };
 }
 
+componentDidMount(){
+  console.log(this.props)
+  this.props.getProductQuestions(3);
+}
+
 render(){
-  return <div><div>QuestionList (map out questions here)</div><Question/></div>
+  return (
+  <div>
+    {
+    this.props.store.questions[0] ? (this.props.store.questions.map( (question, index) => {
```

Comment:
Good logic used here. Good idea we should use going forward on mapping only if the data is present.